Tutorials/PyNetKet/test_tutorials.py

```diff
@@ -0,0 +1,40 @@
+# This files imports all tutorial examples to make them testable
```

Comment:
Oh, thanks for this, before indeed it was a mess!

Reply:
Though I would not recommend running this as part of the regular tests because the tutorials take very long. However, I found it helpful to discover whether a commit breaks the tutorials.